### ISSUE TITLE
sandi.net-boundaries-2014.yaml import error:  ResourceInvalidError: Resource is invalid: /example.com

### ISSUE THREAD

**Issue by nmb10** [CivicKnowledge/ambry#112]
run:
```bash
python load_pre10.py ../../pre-10-bundles/converted/sandi.net-boundaries-2014/sandi.net-boundaries-2014.yaml
```

error:
```
Starting import ../../pre-10-bundles/converted/sandi.net-boundaries-2014/sandi.net-boundaries-2014.yaml...
Loading bundle: sandi.net-boundaries-2014-0.0.1~d03M001
INFO sandi.net-boundaries-2014 ---- Synchronized ----
Starting ingest...
INFO sandi.net-boundaries-2014 Ingesting: example from http://example.com
Traceback (most recent call last):
  File "load_pre10.py", line 251, in <module>
    main()
  File "load_pre10.py", line 220, in main
    _ingest(b)
  File "load_pre10.py", line 177, in _ingest
    b.ingest(force=force, clean_files=clean_files)
  File "/home/nmb10/projects/ambry_project/ambry/bundle/bundle.py", line 873, in ingest
    clean=force, account_accessor=account_accessor)
  File "/home/nmb10/.virtualenvs/ambry/local/lib/python2.7/site-packages/ambry_sources/download.py", line 42, in get_source
    cache_path, download_time = download(spec.url, cache_fs, account_accessor, clean=clean)
  File "/home/nmb10/.virtualenvs/ambry/local/lib/python2.7/site-packages/ambry_sources/download.py", line 170, in download
    cache_fs.remove(cache_path)
  File "/home/nmb10/.virtualenvs/ambry/local/lib/python2.7/site-packages/fs/errors.py", line 257, in wrapper
    return func(self,*args,**kwds)
  File "/home/nmb10/.virtualenvs/ambry/local/lib/python2.7/site-packages/fs/osfs/__init__.py", line 275, in remove
    os.remove(sys_path)
fs.errors.ResourceInvalidError: Resource is invalid: /example.com
```

**Comment by ericbusboom** [comment]
Closed by #122; just set an error, viewable in `ambry list`, and handle the problem manually.

**Comment by ericbusboom** [comment]
Actually, this should just generate an ingestion error, so we can fix it manually later.
